firebase-firestore/src/main/java/com/google/firebase/firestore/CollectionReference.java

```diff
@@ -56,10 +56,10 @@ public String getId() {
   }
 
   /**
-   * Gets a DocumentReference to the document that contains this collection. Only subcollections are
-   * contained in a document. For root collections, returns null.
+   * Gets a {@link DocumentReference} to the document that contains this collection. Only
+   * subcollections are contained in a document. For root collections, returns null.
```

Comment:
When mentioning the `null` literal we should mark that with `{@code}`. See e.g.:

https://docs.oracle.com/javase/7/docs/api/java/io/File.html#getParentFile()